launcher/ui/pages/modplatform/technic/TechnicPage.cpp

```diff
@@ -11,6 +11,11 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
+ *
+ * -----
+ *
+ * This source and all future changes to it are subject to the Microsoft Permissive License (MS-PL).
+ * Please see the COPYING.md file for more information.
```

Comment:
Drop the MS-PL please. I decided to not go through with that in the end.

Reply:
Sure, my QT Creator auto-generated that.